server/preprocessing/other-scripts/text_similarity.R

```diff
@@ -14,6 +14,7 @@ print(params_file)
 setwd(wd) #Don't forget to set your working directory
 
 source(paste("../other-scripts/vis_layout.R", sep=""))
+source('../other-scripts/utils.R')
 
```

Comment:
I think here the altmetrics script is required: `source('../other-scripts/altmetrics.R')`

Reply:
Yes, indeed. Thanks for pointing this out.